truelearn/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""major.minor.patch versioning convention
+==========================================
+major: breaking changes
+minor: new features
+patch: bug fixes
+"""
+__version__ = '1.1.1'
+
+__all__ = ['datasets', 'learning', 'models', 'preprocessing', 'tests', 'utils']
```

Comment:
Let's stick with the double-quoted string as I've used it in the classifier and it makes more sense for someone with a C/C++ background.